src/JSInterop/Microsoft.JSInterop/test/Infrastructure/DotNetDispatcherTest.cs

```diff
@@ -1,6 +1,7 @@
 // Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
+#nullable disable
```

Comment:
Why do we disable it here?

Reply:
We're primarily focused on making nullability work in src projects. Updating tests is nice because it sometimes helps identify code paths that are intended to be null tolerant or explicitly support null values. However, some of these tests are authored poorly and would need extensive rewriting to be correct. It's very little value for the amount of work required to fix the tests.